jujugui/static/gui/src/app/utils/changes-utils.js

```diff
@@ -0,0 +1,242 @@
+            var units = change.command.args[0];
+            changeItem.description = units.length + ' unit' +
+                (units.length === 1 ? ' has' : 's have') +
+                ' been removed from ' + units[0].split('/')[0];
+            break;
+          case '_expose':
+            changeItem.icon = 'changes-service-exposed';
+            changeItem.description = change.command.args[0] + ' exposed';
+            break;
+          case '_unexpose':
+            changeItem.icon = 'changes-service-unexposed';
+            changeItem.description = change.command.args[0] + ' unexposed';
+            break;
+          case '_add_relation':
+            var serviceList = this.getRealRelationEndpointNames(
+                change.command.args, services);
+            changeItem.icon = 'changes-relation-added';
+            changeItem.description = change.command.args[0][1].name +
+                ' relation added between ' +
+                serviceList[0] + ' and ' + serviceList[1] + '.';
+            break;
+          case '_remove_relation':
+            changeItem.icon = 'changes-relation-removed';
+            changeItem.description = change.command.args[0][1].name +
+                ' relation removed between ' +
+                change.command.args[0][0] +
+                ' and ' +
+                change.command.args[1][0] + '.';
+            break;
+          case '_addMachines':
+            var machineType = change.command.args[0][0].parentId ?
+                'container' : 'machine';
+            changeItem.icon = 'changes-' + machineType + '-created';
+            changeItem.description = change.command.args[0].length +
+                ' ' + machineType +
+                (change.command.args[0].length !== 1 ? 's have' : ' has') +
+                ' been added.';
+            break;
+          case '_destroyMachines':
+            /*jshint -W004*/
+            var machineType = change.command.args[0][0].indexOf('/') !== -1 ?
+                'container' : 'machine';
+            changeItem.icon = 'changes-' + machineType + '-destroyed';
+            changeItem.description = change.command.args[0].length +
+                ' ' + machineType +
+                (change.command.args[0].length !== 1 ? 's have' : ' has') +
+                ' been destroyed.';
+            break;
+          case '_set_config':
+            var cfgServ = services.getById(change.command.args[0]);
+            changeItem.icon = 'changes-config-changed';
+            changeItem.description = 'Configuration values changed for ' +
+                cfgServ.get('displayName').match(removeBrackets)[1] + '.';
+            break;
+          default:
+            changeItem.icon = 'changes-service-exposed';
+            changeItem.description = 'An unknown change has been made ' +
+                'to this enviroment via the CLI.';
+            break;
+        }
+      }
+      if (skipTime || !change) {
+        changeItem.time = '00:00';
```

Comment:
Should this not be a `-` or something to show there is no time info. Rather then midnight.